recipes/flake8-junit-report/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "flake8-junit-report" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8890c1ebe0acb516fefacddec4b802bca9f89bb07db933e4ee3cd11ceaa1e8b
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - cp $RECIPE_DIR/test_flake8_file.py $SRC_DIR  # [not win]
+    - copy %RECIPE_DIR%\test_flake8_file.py %SRC_DIR%  # [win]
+    - {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - flake8_junit = junit_conversor.__main__:main
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+  run:
+    - python >=2.7
+    - flake8
+
+test:
+  imports:
+    - junit_conversor
+  source_files:
+    - test_flake8_file.py
```

Comment:
To copy a file from the recipe to the test directory, you should use `files`, not `source_files` in the tests section: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-files

No need to copy that file to the source directory.